libbeat/docs/version.asciidoc

```diff
@@ -1,4 +1,4 @@
-:stack-version: 5.5.0
-:doc-branch: 5.5
+:stack-version: 5.6.0
```

Comment:
That's not set by `set_version`, but it's part of the release checklist to update it. I think it's fine to be changed here.